WebRoot/js/zimbraMail/share/view/dialog/ZmTwoFactorSetupDialog.js

```diff
@@ -437,10 +437,13 @@ function() {
 
 ZmTwoFactorSetupDialog.prototype._handleKeyUp =
 function(ev) {
-	var value = ev && ev.target && ev.target.value && ev.target.value.length;
+	var value = ev && ev.target && ev.target.value;
 	var currentDivId = this._divIds[this._stages[this._stageIndex]][this._divIndexInStage];
-	if (currentDivId === this._passwordDivId || currentDivId === this._codeDivId || currentDivId === this._emailAddressDivId) {
-		this.setButtonEnabled(ZmTwoFactorSetupDialog.NEXT_BUTTON, !!value);
+	if (currentDivId === this._passwordDivId || currentDivId === this._codeDivId) {
+		var valueLength = value && value.length;
+		this.setButtonEnabled(ZmTwoFactorSetupDialog.NEXT_BUTTON, !!valueLength);
```

Comment:
nit: can be reduced to a single line
```suggestion
		this.setButtonEnabled(ZmTwoFactorSetupDialog.NEXT_BUTTON, !!(value && value.length));
```

Reply:
updated. Thank you @silentsakky for your suggestion.